indexing-service/src/test/java/io/druid/indexing/worker/TaskAnnouncementTest.java

```diff
@@ -49,6 +49,10 @@ public TaskAnnouncementTest()
   {
     TestUtils testUtils = new TestUtils();
     jsonMapper = testUtils.getTestObjectMapper();
+
+    final File baseDir = new File("lol");
+    baseDir.mkdir();
+    baseDir.deleteOnExit();
```

Comment:
It was for checking that the directory exists in the constructor of LocalFirehoseFactory. Removed now.